e2e/utils/TestUtils.ts

```diff
@@ -254,22 +254,32 @@ export async function tool(tool: string, ...args: string[]): Promise<string> {
   }
 }
 
-export async function waitForRestartVM(progressBar: Locator, options = { timeout: 10_000, interval: 200 }): Promise<void> {
+export async function waitForRestartVM(progressBar: Locator, options = { timeout: -1, interval: 200 }): Promise<void> {
```

Comment:
Why the change to the `timeout` default? I don't see any places that use `waitForRestartVM` that specify something other than the default for `timeout`. And we just set it to a new default below.

Reply:
Seeing how nothing passes `options` to `waitForRestartVM`, let's just drop the options argument.